data/aadii0408.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Aaditya Champaneri || Aadi ",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Artist By Choice, Engineer By Default | Python Developer",
+  
+  "tags": [ "Python", "Machine Learning", "Open Source", "Marketing", "Communities"],
+  
+  "avatar": "https://github.com/aadii0408.png",
```

Comment:
hey @aadii0408 we don't need avatar in new update you can remove it

Reply:
```suggestion
```